src/index.js

```diff
@@ -251,6 +258,13 @@ export default class Swup {
 		}
 	}
 
+	triggerWillOpenNewWindow(triggerEl) {
```

Comment:
Why are you using `triggerWillOpenNewWindow` instead of `linkWillOpenNewWindow` here?

Reply:
I'm trying to think ahead and make this usable for `form`s and the form plugin as well. Is this just making it confusing?

Reply:
No, that's what I was presuming. Just wanted to make sure. Looking good, will approve this now!